improver/cube_combiner.py

```diff
@@ -170,11 +170,14 @@ def process(self, cube_list, new_diagnostic_name, use_midpoint=False):
         if self.operation == 'mean':
             result.data = result.data / len(cube_list)
 
-        # update any coordinates that have been expanded, and rename output
-        expanded_coord_names = self._get_expanded_coord_names(cube_list)
-        if expanded_coord_names:
-            result = expand_bounds(result, cube_list, expanded_coord_names,
-                                   use_midpoint=use_midpoint)
+        # where the operation is "multiply", retain all coordinate metadata
+        # from the first cube in the list; otherwise expand coordinate bounds
```

Comment:
I was just wondering, as noted here that all coordinate metadata from the first cube in the list is retained, whether the docstring for the `CubeCombiner` plugin or the `combine` CLI should mention that the metadata will be taken from the first cube in the list. I might think that it might cause confusion, if someone doesn't realise that the order of multiplication impacts the metadata. What do you think?

Reply:
Good idea.  Added to both plugin and CLI.